scripts/buildCSS.js

```diff
@@ -25,5 +25,5 @@ if (!fs.existsSync(dir)) {
   fs.mkdirSync(dir);
 }
 
-const outputFilePath = optimizeForProduction ? `${dir}/rheostat.css` : `${dir}/rheostat.css`;
```

Comment:
These were the same? The logic is already here: https://github.com/airbnb/rheostat/pull/244/files#diff-86fcfa6a3dc885f28527e9ff85639ed9R22